linera-service/tests/end_to_end_tests.rs

```diff
@@ -1059,9 +1070,8 @@ async fn test_reconfiguration(network: Network) {
     client.transfer(5, chain_2, chain_1).await;
 
     // Restart last server (dropping it kills the process)
-    let validator_4 = local_net.get_mut(3).unwrap();
-    validator_4.kill_server(3);
-    validator_4.add_server(runner.run_server(4, 3).await);
+    runner.kill_server(4, 3);
+    runner.restart_server(4, 3).await;
```

Comment:
Maybe it should either be called `start_server`, or it should also kill the existing one.